libs/platform/ebpf_platform.h

```diff
@@ -1191,7 +1191,7 @@ extern "C"
      * @return EBPF_INVALID_ARGUMENT The algorithm is not supported.
      */
     _Must_inspect_result_ ebpf_result_t
-    ebpf_cryptographic_hash_create(_In_z_ const wchar_t* algorithm, _Outptr_ ebpf_cryptographic_hash_t** hash);
+    ebpf_cryptographic_hash_create(_In_ const ebpf_utf8_string_t* algorithm, _Outptr_ ebpf_cryptographic_hash_t** hash);
```

Comment:
Update line 1187 to say it must be one of the values at https://learn.microsoft.com/en-us/windows/win32/seccng/cng-algorithm-identifiers
Also there's a grammar mismatch issue there with "values are" being used with a singleton.